### PR TITLE
Notifications check

### PR DESCRIPTION
**Pull request by Nesze**

### CODE REVIEW

File: publishCheck.go [hunk 1]
```diff
@@ -91,10 +105,42 @@ func (c ContentCheck) isCurrentOperationFinished(pc PublishCheck, response *http
...
+	}
+
+	return strings.Contains(string(data), pc.Metric.UUID)
+}
+
+func (n NotificationsCheck) buildURL(pm PublishMetric) string {
+	base := pm.endpoint.String()
+	queryParam := url.Values{}
+	//e.g. since=2015-07-23T00:00:00.000Z
+	since := time.Now().Add(-2 * time.Minute).Format(time.RFC3339)
```
Comment (catalin-ilea) [inline publishCheck.go]: Shouldn't you use pm.publishDate?

File: publishCheck.go [hunk 2]
```diff
@@ -42,14 +50,20 @@ func NewPublishCheck(pm PublishMetric, t int, ci int, rs chan PublishMetric) *Pu
 // Returns true if the content is available at the endpoint, false otherwise.
 func (pc PublishCheck) DoCheck() bool {
 	info.Printf("Running check for UUID [%v]\n", pc.Metric.UUID)
-	resp, err := http.Get(pc.Metric.endpoint.String() + pc.Metric.UUID)
+	check := endpointSpecificChecks[pc.Metric.config.Alias]
+
+	//TODO remove this after the notification service gets rewritten
+	if pc.Metric.config.Alias == "notifications" {
+		time.Sleep(120 * time.Second)
```
Comment (catalin-ilea) [inline publishCheck.go]: You should sleep 120 - timeSincePublish

Comment (tamas-molnar) [inline publishCheck.go]: Shouldn't the scheduler be responsible for scheduling checks? Maybe set the granularity to 1 for notifications checks, see if that works.

File: publishCheck.go [hunk 3]
```diff
@@ -42,14 +50,20 @@ func NewPublishCheck(pm PublishMetric, t int, ci int, rs chan PublishMetric) *Pu
 // Returns true if the content is available at the endpoint, false otherwise.
 func (pc PublishCheck) DoCheck() bool {
 	info.Printf("Running check for UUID [%v]\n", pc.Metric.UUID)
-	resp, err := http.Get(pc.Metric.endpoint.String() + pc.Metric.UUID)
+	check := endpointSpecificChecks[pc.Metric.config.Alias]
+
+	//TODO remove this after the notification service gets rewritten
```
Comment (tamas-molnar) [inline publishCheck.go]: Please open a GH ticket for this, maybe it's more easy to track, who knows when the notification service will be rewritten.

File: publishCheck.go (hunk 1 above)
Comment (tamas-molnar) [inline publishCheck.go]: +1

File: publishCheck.go (hunk 3 above)
Comment (Nesze) [inline publishCheck.go]: Done. https://github.com/Financial-Times/publish-availability-monitor/issues/38

**Comment by tamas-molnar** [comment]
lgtm

**Comment by catalin-ilea** [comment]
lgtm

**Pull request merged**